### PR TITLE
docs(github): add contribution and release instructions

### PR DESCRIPTION
**Pull request by KevinGDialpad**
Mostly copied and adapted from gcloud-aio

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 RESTful Google Cloud Client Library for Python
 ==============================================
 
 This project is a collection of Google Cloud client libraries for the REST-only
-APIs; its *raison d'etre* is to implement a simple `CloudTasks API`_ as well as
+APIs; its *raison d'être* is to implement a simple `CloudTasks API`_ as well as
 a more abstract TaskManager.
 
+If you don't need to support Python 2, you probably want to use `gcloud-aio`_,
+which has support for all the same APIs (and more!) and additionally includes support for ``asyncio``.
```
Comment (TheKevJames) [inline README.rst]: nit: please word-wrap this to 80 characters.

**Comment by KevinGDialpad** [comment]
I'm investigating the error... Adding quotes around `${GOOGLE_APPLICATION_CREDENTIALS}` gave me this error:
```
====>> echo ${GOOGLE_SERVICE_JSON} | base64 -d > "${GOOGLE_APPLICATION_CREDENTIALS}"
  #!/bin/bash -eo pipefail
echo ${GOOGLE_SERVICE_JSON} | base64 -d > "${GOOGLE_APPLICATION_CREDENTIALS}"
/bin/bash: : No such file or directory
Error: Exited with code 1
Step failed
```
which made me think there is something wrong with the Docker image. I noticed that gcloud-aio uses `thekevjames/nox:2018.10.17` instead of `googleapis/nox:0.17.0` so I'm trying to use that instead.

I'll push again if/when it works.

**Comment by TheKevJames** [comment]
I suspect the error is due to not having that second environment variable set, like we do in [gcloud-aio](https://github.com/talkiq/gcloud-aio/blob/33e2055a1c1d40ff57f7fcdc41dc72038e073aa2/.circleci/config.yml#L11). I'm going to merge this PR as-is; looks like I need to do an update of the assigned permissions anyway, so I'll handle these at the same time. Thanks for noticing the issue!

**Comment by KevinGDialpad** [comment]
Yep that's what I was getting at.
After fixing that, I got `/bin/bash: git: command not found` on the next line, so that might not be the end of it.

**Comment by KevinGDialpad** [comment]
And FYI - I also have a [branch](https://github.com/KevinGDialpad/gcloud-rest/tree/python3.7) where I'm fixing nox and adding tests for Python 3.7. I'll create a PR when I can make CircleCI run nox properly.

**Comment by TheKevJames** [comment]
Ah, I think you and I had the same idea. I've basically got #33 building properly at this point; I'll hold off duplicating any of your work in adding/fixing tests and merge it as-is.

**Comment by TheKevJames** [comment]
```
nox > Command pip install -e . failed with exit code 1:
Obtaining file:///root/project
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/root/project/setup.py", line 8, in <module>
        README = f.read()
      File "/root/project/.nox/unit_tests-3-5/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 195: ordinal not in range(128)
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /root/project/
```

I think your change for spelling `etre` correctly may have broken the Python 3.5 / 3.6 installation, FYI. Damn unicode problems! :(

**Comment by KevinGDialpad** [comment]
Ahaha oops. I was seeing the same errors - good to know the fix is simple.

I don't want to dig into this now but it's disappointing to see that it's trying to decode all the files using ASCII by default... Oh well.

**Comment by TheKevJames** [comment]
So `open` tries to open files as ascii by default, so the call to `.read()` tries to parse it as such -- if we `open`'d as `bytes` instead, we could set `README: bytes = f.read()` perfectly fine -- note though that we'd need to `.decode()` it according to the correct character set in order to pass that value into `setuptools.setup()`; I don't know offhand what character code points are accepted there (or even further downstream, eg. what can be rendered by tools which show package descriptions?)

**Comment by KevinGDialpad** [comment]
Oh my bad; I didn't notice it was in setup.py and thought it was somewhere upstream. Please disregard my comment :)

It would be an interesting problem to fix (or know how to fix) but let's not get into that because of a random French person on the team :)

**Pull request merged**